README.md

```diff
@@ -40,6 +40,12 @@ cAdvisor also exposes container stats as [Prometheus](http://prometheus.io) metr
 
 [Heapster](https://github.com/kubernetes/heapster) enables cluster wide monitoring of containers using cAdvisor.
 
+## Axibase Time Series Database
+
+[Axibase Time Series Database](http://axibase.com/products/axibase-time-series-database/) can collect Docker metrics through cAdvisor for long-term retention, analytics and visualization. A single ATSD instance can collect metrics from many Docker hosts and cAdvisors.
+
+See the [documentation](/docs/atsd_install.md) for more information.
+
```

Comment:
I don't think the main README is the right place to describe every storage option. I think we should add a new directory in the docs folder called `storage`, with a README that includes snippets like this one, as well as more detailed documents (such as atsd_install.md).